README.md

```diff
@@ -65,6 +59,31 @@ This test doesn't contain the default theme. You have to install the theme you w
 - i18n
 - Post share
 - SEO
+- RSS [Autodiscovery](http://www.rssboard.org/rss-autodiscovery) support
+  * Example:
+  ``` html
+  <link rel="alternate" href="path/of/rss" type="application/atom+xml">
+  ```
+  * Some RSS plugins (e.g. [hexo-generator-feed](https://github.com/hexojs/hexo-generator-feed) 2.1+) insert autodiscovery by default. There is a slight performance benefit if a theme inserts it, instead of the plugin. To take advantage of that, autodiscovery needs to be disabled in the plugin.
+    ``` yml
+    feed:
+      autodiscovery: false
+    ```
+  * RSS plugin could generate more than one type of RSS (e.g. Atom & RSS2). Here is an example EJS snippet for multi-format support:
+    ``` js
+    <% if (config.feed) { %>
+      <% if (config.feed.type.length && config.feed.path.length) { %>
+        <% if (typeof config.feed.type === 'string') { %>
+          <link rel="alternate" type="application/<%- config.feed.type.replace(/2$/, '') %>+xml" title="<%= config.title %>" href="<%- url_for(config.feed.path) %>">
+        <% } else { %>
+          <% for (const i in config.feed.type) { %>
+            <link rel="alternate" type="application/<%- config.feed.type[i].replace(/2$/, '') %>+xml" title="<%= config.title %>" href="<%- url_for(config.feed.path[i]) %>">
+          <% } %>
+        <% } %>
+      <% } %>
+    <% } %>
+    ```
```

Comment:
IMHO, we should bring up a `feed_autodiscovery_tag()` helper instead of an `ejs` example.

Reply:
just realized we already have one: https://hexo.io/docs/helpers#feed-tag